src/gradualizer_cli.erl

```diff
@@ -101,7 +114,7 @@ parse_opts([A | Args], Opts) ->
         "--stop-on-first-error"    -> parse_opts(Args, [stop_on_first_error | Opts]);
         "--no-stop-on-first-error" -> parse_opts(Args, [{stop_on_first_error, false} | Opts]);
         "--crash-on-error"         -> parse_opts(Args, [crash_on_error | Opts]);
-        "--no-crash-on-error"      -> parse_opts(Args, [{no_crash_on_error, false} | Opts]);
+        "--no-crash-on-error"      -> parse_opts(Args, [{crash_on_error, false} | Opts]);
```

Comment:
Nice catch 😀

Reply:
Yes, the tests helped me find it, believe it or not. :-)

Reply:
Tests that actually help? Amazing!